src/momento/aio/_scs_data_client.py

```diff
@@ -35,21 +38,17 @@ def _make_metadata(cache_name: str) -> Metadata:
 class _ScsDataClient:
     """Internal"""
 
-    def __init__(
-        self,
-        auth_token: str,
-        endpoint: str,
-        default_ttl_seconds: int,
-        operation_timeout_ms: Optional[int],
-    ):
+    def __init__(self, configuration: Configuration, credential_provider: CredentialProvider, default_ttl: timedelta):
+        endpoint = credential_provider.get_cache_endpoint()
         self._logger = logs.logger
         self._logger.debug("Simple cache data client instantiated with endpoint: %s", endpoint)
-        self._default_deadline_seconds = (
-            _DEFAULT_DEADLINE_SECONDS if not operation_timeout_ms else operation_timeout_ms / 1000.0
+        default_deadline: timedelta = (
+            configuration.get_transport_strategy().get_grpc_configuration().get_deadline() or _DEFAULT_DEADLINE_SECONDS
```

Comment:
This suggests a wrapper method to hide all that and to provide a default.

Reply:
Added a utility method to DRY this calculation out, as it is used by both sync and async clients.